app/internal/emotion.py

```diff
@@ -0,0 +1,64 @@
+import text2emotion as te
+from typing import List, Union
+
+from app.config import (
+    CONTENT_WEIGHTS,
+    LEVEL_OF_SIGNIFICANCE,
+    TITLE_WEIGHTS)
+
+
+EMOTIONS = [("Happy", "&#128515"),
+            ("Sad", "&#128577"),
+            ("Angry", "&#128544"),
+            ("Fear", "&#128561"),
+            ("Surprise", "&#128558")]
+
+
+def dominant_emotion(title: str, content: str) -> Union[List]:
+    """
+    get text from event title and content and return
+    the dominant emotion, emotion score and smiley code
+    """
+    dominant_emotion = [None, 0, None]
+    duplicate_dominant_flag = 0
+    title_emotion = te.get_emotion(title)
```

Comment:
Prefer to choose on, as te.get_emotion might be a heavy computing calculation

Reply:
could you explain what do you mean to choose on?
thanks!

Reply:
Probably meant "prefer to put on the `else` clause", but now I see you need them both ^^"